testsuite/integration/web/pom.xml

```diff
@@ -76,6 +76,11 @@
             <artifactId>bcpkix-jdk15on</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.bouncycastle</groupId>
+            <artifactId>bcutil-jdk15on</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
I haven't found any test class requiring it. @boris-unckel Could you describe the reason why you are adding it here? I think it could be unnecessary

Reply:
Hello @yersan I have added it after the [exclusion on bcpkix-jdk15on](https://github.com/wildfly/wildfly-core/pull/4628/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8R1313-R1318) in wfcore because it is transient depending on [org.bouncycastle/bcpkix-jdk15on/1.69](https://mvnrepository.com/artifact/org.bouncycastle/bcpkix-jdk15on/1.69) In wfcore the full integration build fails at SSL/Elytron/Mail tests, unfortunately with a different message, i.E. 
```
org.wildfly.test.integration.elytron.certs.crl.CertificateRevocationListTestCase.testTwoWayCertificateNotInMultipleServerCrls: java.lang.AssertionError: Failed to execute line 'run-batch'
...
Operation: /subsystem=undertow/server=default-server/https-listener=https:write-attribute(name=ssl-context,value=otherServerSslContext)
Failure: WFLYCTL0369: Required capabilities are not available:
    org.wildfly.security.ssl-context.otherServerSslContext; Possible registration points for this capability: 
  /subsystem=elytron/server-ssl-context=*
  /subsystem=elytron/client-ssl-context=*
  /subsystem=elytron/server-ssl-sni-context=*
...
```

Reply:
OK, I have removed it locally already. I'll wait for the push until PR [https://github.com/wildfly/wildfly/pull/14398](https://github.com/wildfly/wildfly/pull/14398) is merged.